### ISSUE TITLE
Article items for translation

### ISSUE THREAD

**Issue by DDinghoya** [xjsxjs197/WiiSXRX_2022#194]
I looked for items that were not translated or could not be printed on Wiistation screen.

**1. Missing items as below;**
msgid "Gun/Mouse"
msgid "Menu:"
msgid "Limit FPS:"
msgid "Gun/Mouse"
msgid "D-Up"
msgid "D-Down"
msgid "D-Right"
msgid "D-Left"
msgid "Normarl Y"

![01](https://github.com/xjsxjs197/WiiSXRX_2022/assets/19677686/8436ac2d-6074-4108-89fb-7b62efe96bb6)

**2. Missing items as below;**
msgid "Multitap 1"
msgid "Multitap 2"

![02](https://github.com/xjsxjs197/WiiSXRX_2022/assets/19677686/7bb44c95-6bb5-4c6d-9735-7830a2c336fa)

**3. Missing items as below;**
msgid "Analog" : _Translated but not applied_
msgid "Lightgun"

![03](https://github.com/xjsxjs197/WiiSXRX_2022/assets/19677686/292f5e40-3cbe-4de7-9865-e8bb7c82b0c2)

**4. Missing items as below;**
msgid "Interlaced" : _Translated but not applied_
msgid "Trap" : _Missing & Translated but text cut off_ Print out "트" -> "트랩" is correct.
msgid "Deflicker" : _Translated but not applied_

![04](https://github.com/xjsxjs197/WiiSXRX_2022/assets/19677686/fbeae2aa-314a-466f-8cd3-58b33acb8db7)

**Comment by xjsxjs197** [comment]
That's true, but I'm not very good at translating for multiple languages.
It would be great if there were domestic personnel to translate it.

**Comment by DDinghoya** [comment]
> That's true, but I'm not very good at translating for multiple languages.
> It would be great if there were domestic personnel to translate it.

Yes, You're right! I'm not asking you to translate.

As you know, I compleated translation for Korean. But, As mentioned above, many recent commits have left many lang items missing from translation. :)
